src/server/clientiface.cpp

```diff
@@ -244,6 +245,13 @@ void RemoteClient::GetNextBlocks (
 	s32 nearest_sent_d = -1;
 	//bool queue_is_full = false;
 
+	// add modified blocks to the beginning of the queue
+	g_profiler->avg("Server::modified blocks [#]", m_blocks_modified.size());
```

Comment:
will this number be meaningful? since it's the average of the send loop of *all* clients